### PR TITLE
Edited PSR-16 files for small typos/grammar fixes.

### PR DESCRIPTION
**Pull request by DocFX**
Only non-structural changes, just a few punctuation/wording/grammar fixes.

### CODE REVIEW

File: accepted/PSR-16-simple-cache.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ own caching implementations and easily rely on the one given to them by the
 framework, or another dedicated cache library.
 
 PSR-6 solves this problem already, but in a rather formal and verbose way for
-what the most simple use cases need. This simpler approach aims to build a
-standardized streamlined interface for common cases.  It is independent of
+what the most simple use cases need. This simpler approach aims at building a
+standardized streamlined interface for common cases. It is independent of
```
Comment (Crell) [inline accepted/PSR-16-simple-cache.md]: "aims to build" is correct grammar in English.

File: accepted/PSR-16-simple-cache.md [hunk 2]
```diff
@@ -70,8 +70,8 @@ by an integer representing time in seconds, or a DateInterval object.
 cached item. Implementing libraries MUST support keys consisting of the
 characters `A-Z`, `a-z`, `0-9`, `_`, and `.` in any order in UTF-8 encoding and a
 length of up to 64 characters. Implementing libraries MAY support additional
-characters and encodings or longer lengths, but must support at least that
-minimum.  Libraries are responsible for their own escaping of key strings
+characters and encodings or longer lengths, but MUST support at least that
+minimum. Libraries are responsible for their own escaping of key strings
```
Comment (Crell) [inline accepted/PSR-16-simple-cache.md]: This one is tricky; changing `must` to `MUST` changes the meaning from an informal requirement to a spec-formal requirement.  On the other hand, it's very clear from context (and from PSR-6, for which I was the editor, which shares this text) that `MUST` is the intent here.

So... I'm not sure.  I'm probably OK here?

File: accepted/PSR-16-simple-cache.md [hunk 3]
```diff
@@ -97,18 +97,18 @@ Implementing libraries MUST support all serializable PHP data types, including:
 *    **Strings** - Character strings of arbitrary size in any PHP-compatible encoding.
 *    **Integers** - All integers of any size supported by PHP, up to 64-bit signed.
 *    **Floats** - All signed floating point values.
-*    **Boolean** - True and False.
+*    **Booleans** - True and False.
```
Comment (Crell) [inline accepted/PSR-16-simple-cache.md]: Legit.

File: accepted/PSR-16-simple-cache.md [hunk 4]
```diff
@@ -97,18 +97,18 @@ Implementing libraries MUST support all serializable PHP data types, including:
...
 *    **Null** - The null value (although it will not be distinguishable from a
 cache miss when reading it back out).
 *    **Arrays** - Indexed, associative and multidimensional arrays of arbitrary depth.
-*    **Object** - Any object that supports lossless serialization and
+*    **Objects** - Any object that supports lossless serialization and
```
Comment (Crell) [inline accepted/PSR-16-simple-cache.md]: Legit.

File: accepted/PSR-16-simple-cache.md [hunk 5]
```diff
@@ -155,7 +155,7 @@ interface CacheInterface
      * Persists data in the cache, uniquely referenced by a key with an optional expiration TTL time.
      *
      * @param string                 $key   The key of the item to store.
-     * @param mixed                  $value The value of the item to store, must be serializable.
+     * @param mixed                  $value The value of the item to store. Must be serializable.
```
Comment (Crell) [inline accepted/PSR-16-simple-cache.md]: Legit.

File: accepted/PSR-16-simple-cache.md [hunk 6]
```diff
@@ -235,7 +235,7 @@ interface CacheInterface
      * NOTE: It is recommended that has() is only to be used for cache warming type purposes
      * and not to be used within your live applications operations for get/set, as this method
      * is subject to a race condition where your has() will return true and immediately after,
-     * another script can remove it making the state of your app out of date.
+     * another script can remove it, making the state of your app out of date.
```
Comment (Crell) [inline accepted/PSR-16-simple-cache.md]: Legit.

File: accepted/PSR-16-simple-cache.md [hunk 7]
```diff
@@ -274,8 +274,8 @@ namespace Psr\SimpleCache;
 /**
  * Exception interface for invalid cache arguments.
  *
- * When an invalid argument is passed it must throw an exception which implements
- * this interface
+ * When an invalid argument is passed, it must throw an exception which implements
+ * this interface.
```
Comment (Crell) [inline accepted/PSR-16-simple-cache.md]: The period on the end it Legit.

The comma I would argue is unnecessary.

**Comment by Crell** [comment]
As with the PSR-13 notes, please don't mess with the single vs double space after periods. Don't open that can of worms. :smile:

File: accepted/PSR-16-simple-cache.md (hunk 1 above)
Comment (DocFX) [reply accepted/PSR-16-simple-cache.md]: I actually saw more of the other one. Both are correct and yet I might be influenced by sports and video games, for "aim at" seems to be used for physical targeting slightly more than the other way of writing it. Might remove that from the PR.

File: accepted/PSR-16-simple-cache.md (hunk 2 above)
Comment (DocFX) [reply accepted/PSR-16-simple-cache.md]: That's exactly what I thought while reading it. I read the sentence then I thought "Hey this is part of the recommendation, right?". Also, it's following another part of the sentence that's strongly linked to it and that uses a capitalized version as a start. Again, I proposed this for cohesion only.

**Comment by DocFX** [comment]
I promise I'll take a look at other PSR and NOT edit the double spaces if I find any, @Crell :)

File: accepted/PSR-16-simple-cache.md (hunk 2 above)
Comment (Jean85) [reply accepted/PSR-16-simple-cache.md]: I agree, this seemed like the original intent. I'm :+1: on this, any library doing otherwise is breaking interop.

**Pull request merged**